### PR TITLE
feat: add binary install method

### PR DESCRIPTION
**Pull request by jowparks**
### What changed?

Added binary install page

Screenshot 2024-01-09 at 1 12 50 PM

---

<sub>**Reminder:** If content (docs, blogs, pages) is moved or renamed, please ensure there are no broken links.</sub>

### CODE REVIEW

**Comment by NullSoldier** [comment]
Usually this is called Standalone Executable

**Comment by jowparks** [comment]
> Usually this is called Standalone Executable on most sites when you install their thing, or Portable Executable.

Sounds good I will rename

**Review by NullSoldier — APPROVED** [review]

File: content/get-started/install-standalone.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+1. Search for 'Environment Variables' and open 'Edit the system environment variables'.
+2. Click 'Environment Variables', then under 'System variables', select 'Path' and 'Edit'.
+3. Click 'New' and add the path to your Iron Fish Standalone Executable.
+4. Click 'OK' to save.
+
+
+## Updating
+
+Repeat the process above for the newest version of the Iron Fish Standalone Executable.
```
Comment (NullSoldier) [inline content/get-started/install-standalone.mdx:50]: Is this saying you also need to update the path again after you download it? Should this just be download and replace with the latest release in install?

Comment (jowparks) [reply content/get-started/install-standalone.mdx:50]: Yea good call, I will update to say your wording it is better and more explicit.

**Pull request merged**